src/redux/auth/saga.ts

```diff
@@ -55,21 +55,20 @@ export function* loginSaga({ payload: { email, password } }: LoginAction) {
 }
 
 export function* signupSaga({
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
I would say you can just remove the unused property and also the `eslint-disable` comment for now (and use it once it's needed).